packages/xstate-wallet/public/index.html

```diff
@@ -1,7 +1,31 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <% if (process.env.NODE_ENV === 'production') { %>
+    <% if (process.env.WINDOW_CRYPTO_POLYFILL) { %>
+    <script src="https://unpkg.com/window-crypto"></script>
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/web3/1.2.11/web3.min.js"
+      integrity="sha512-6lf28FmolQdo4ap44cXw7j+thYEZzS1/kXxDkX1ppO//SHCageBS1HmYdqkkL819WfGtQ+7TE+uXEwsxjJXEKQ=="
+      crossorigin="anonymous"
+    ></script>
+    <script>
+      window.web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'));
+      window.ethereum = window.web3.currentProvider;
+      window.ethereum.enable = () =>
+        new Promise(r => {
+          console.log('window.ethereum.enable() was called');
+          web3.eth.getAccounts().then(lst => {
+            web3.eth.defaultAccount = lst[0];
+            console.log('Using address ' + web3.eth.defaultAccount);
+            window.ethereum.selectedAddress = web3.eth.defaultAccount;
+            r([window.ethereum.selectedAddress]);
+          });
+        });
+      window.ethereum.networkVersion = 9001;
+      window.ethereum.on = () => {};
+      window.ethereum.enable();
```

Comment:
Maybe it should have its own configuration option, and not re-use `WINDOW_CRYPTO_POLYFILL`? e.g., `INJECT_GANACHE_WEB3_POLYFILL`?